FreeRTOS_TCP_IP.c

```diff
@@ -3844,13 +3844,19 @@
 
         #if ( ipconfigSUPPORT_SELECT_FUNCTION == 1 )
             {
+		
+		#if ( ipconfigDONOT_COPY_SOCKETSET_TO_CHIELD_SOCKET == 1 )
+				pxNewSocket->pxSocketSet = NULL;
```

Comment:
If you are adding a different socket set to the child sockets then this would anyways get overwritten. Did I understand that correctly: _In our a Application, we doesn't need the copy of the SocketSet to the ChildSockets. then ChildSockets works with different socketset then the Parents Sockent_.?

If that is the case, then would you mind explaining how assigning the parent's socket set would matter?

If that is not the case - can you help me understand what you mean?

Thank you,

Reply:
We use FreeRTOS in a Software, which runs on Embedded  with FreeRTOS and on Windows. In Windows, the Accept doesn't copy the SocketSet to the child socket.
Generall it's works in this way:
1. Select with SocketSet, Accept
2. ChildSocket goes in a other Task, where Select, with recived and Send is done.

I would prefere, that this case runs more, like the Socket of Windows and Linux